apps/ios/src/FluentUITester.xcodeproj/project.pbxproj

```diff
@@ -31,6 +33,10 @@
 		9F3A29F4244F657400A401D2 /* Debug.xcconfig */ = {isa = PBXFileReference; lastKnownFileType = text.xcconfig; path = Debug.xcconfig; sourceTree = "<group>"; };
 		9F3A29F5244F658400A401D2 /* Release.xcconfig */ = {isa = PBXFileReference; lastKnownFileType = text.xcconfig; path = Release.xcconfig; sourceTree = "<group>"; };
 		9F986B74244F5EAC0068CA9B /* Dummy.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = Dummy.swift; sourceTree = "<group>"; };
+		9FAFEC8F248888F2005B3023 /* FRNAvatarViewManager.h */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.h; path = FRNAvatarViewManager.h; sourceTree = "<group>"; };
+		9FAFEC9024888901005B3023 /* FRNAvatarViewManager.m */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.objc; path = FRNAvatarViewManager.m; sourceTree = "<group>"; };
+		9FAFEC9224888921005B3023 /* FRNAvatar.m */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.objc; path = FRNAvatar.m; sourceTree = "<group>"; };
+		9FAFEC942488892C005B3023 /* FRNAvatar.h */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.h; path = FRNAvatar.h; sourceTree = "<group>"; };
```

Comment:
These aren't test files so shouldn't be in FluentUITester.xcodeproj or in the apps folder?